src/components/v2/appDetails/appDetails.store.ts

```diff
@@ -0,0 +1,135 @@
+import { BehaviorSubject } from 'rxjs';
+import { ApplicationObject, iNode } from "./appDetails.type";
+import { URLS } from "../../../config";
+
+let applicationObjectTabsSubject: BehaviorSubject<Array<ApplicationObject>> = new BehaviorSubject([] as Array<ApplicationObject>);
+let _nodeTreeActiveParentNode: iNode
+let _nodeTreeActiveNode: iNode
+let _maxTabAllowd = 6
+
+const addAOT = (tabName: string, tabUrl: string, isSelected: boolean, title?: string) => {
+    let tab = {} as ApplicationObject
+    tab.name = tabName
+    tab.url = tabUrl
+    tab.isSelected = isSelected
+    tab.title = title || tabName
+    return tab
+}
+
+export const AppDetailsTabs = {
+    k8s_Resources: "K8s Resources",
+    log_analyzer: "Log Analyzer"
+}
+
+const AppDetailsStore = {
+    getAppDetailsTabs: () => {
+        return applicationObjectTabsSubject.getValue()
+    },
+    getAppDetailsTabsObservable: () => {
+        return applicationObjectTabsSubject.asObservable()
+    },
+    initAppDetailsTabs: (_url: string, displayLogAnalyzer: boolean, isLogAnalyserURL: boolean) => {
+        let aots = [] as Array<ApplicationObject>
+
+        aots.push(addAOT(AppDetailsTabs.k8s_Resources, _url + "/" + URLS.APP_DETAILS_K8, !isLogAnalyserURL))
```

Comment:
strip trailing /, else behaviour is unpredictable in case some adds trailing / to details for eg http://localhost:3000/chart-store/deployments/111/env/7/details/